public/_redirects

```diff
@@ -71,4 +71,4 @@ https://www.asyncapi.io/* https://www.asyncapi.com/:splat 301!
 # TOOLS-REDIRECTION:END
 
 # Additional redirection
-https://www.asyncapi.com/community/meetings https://www.asyncapi.com/community/events 301!
\ No newline at end of file
+https://deploy-preview-1949--asyncapi-website.netlify.app/community/meetings https://deploy-preview-1949--asyncapi-website.netlify.app/community/events 301!
```

Comment:
@vishvamsinh28 It should be pointing to the section on `asyncapi.com` not on any `deployment-preview-xxxx`